### ISSUE TITLE
Close tab button is not centered

### ISSUE THREAD

**Issue by alexdima** [microsoft/vscode#122656]
When hovering with the mouse over the `x`, the rectangle around the `x` is not centered left-right in the whitespace, it touches the right side of the tab while leaving a large gap to the left:

![image](https://user-images.githubusercontent.com/5047891/116541244-82ca2980-a8eb-11eb-9746-038fced237b8.png)

**Comment by bpasero** [comment]
@alexdima do you use any tab related settings that are non default?

**Comment by alexdima** [comment]
You're right! This reproduces only with:
```
"workbench.editor.tabSizing": "shrink"
```

**Comment by bpasero** [comment]
I suspect a regression from the hover feedback.

**Comment by joaomoreno** [comment]
It's also possible that https://github.com/microsoft/vscode/issues/49382 also caused some regressions here.

@bpasero Is it normal that togglign the setting makes the tabs jump, even when there's more than enough space available?

Will fix the action.

**Comment by bpasero** [comment]
Yes tabs can jump because they no longer have a minimal width in that case.
